src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,43 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+
+  constructor(name) {
+    this.health = 100;
+    this.name = name;
+    Animal.addAlive(this);
+  }
+
+  static addAlive(animal) {
+    this.alive.push(animal);
+  }
+
+  static removeDead() {
+    this.alive = this.alive.filter((elem) => elem.health > 0);
```

Comment:
use `animal` instead of `elem` for better readability
you can skip `()`

`this.alive.filter(animal => animal.health > 0);`

Reply:
@mrkovaliv OK, but `()` is set by the linter during tests.